projects/sbb-esta/angular-public/src/lib/usermenu/_usermenu.scss

```diff
@@ -31,17 +36,20 @@ $borderActive5K: solid toPx(4 * $scalingFactor5k) $sbbColorGrey;
   border-top: $borderActive;
   border-right: $borderActive;
   border-left: $borderActive;
+  border-bottom: $borderActive;
```

Comment:
What is the reason for adding this? I see no difference in testing?
(Also replace with `border: $borderActive;` if this is required?)
This also applies to the border-bottom lines below.

Reply:
If you do not put this, When the curtain goes up the bottom edge is missing.

![usermenu-up-issue](https://user-images.githubusercontent.com/47629809/63412689-86a86780-c3f8-11e9-8a75-6692b9c88a04.PNG)

Reply:
Good catch. Please reduce to `border: $borderActive;`. Also apply for the changes below.